include/avif/avif.h

```diff
@@ -1160,6 +1160,11 @@ typedef struct avifDecoder
     // Can be useful to decode the gain map image only.
     avifBool ignoreColorAndAlpha;
 #endif
+
+    // This is true when avifDecoderParse() detects an image sequence track in the image. If this is true, the image can be
+    // decoded either as an animated image sequence or the primary image item can be retrieved as a still image by setting
```

Comment:
Nit: The "either" and "or" parts of this sentence are not of the same form.

The "either" part is "as an animated image sequence".

The "or" part is "the primary image item can be retrieved as a still image".

A possible fix is the following:

> If this is true, the image can be decoded either as an animated image sequence or as a still image (the primary image item) by setting avifDecoderSetSource to the appropriate source.

Reply:
Another possible fix is to move "either" earlier:

> If this is true, either the image can be decoded as an animated image sequence or the primary image item can be retrieved as a still image by setting avifDecoderSetSource to the appropriate source.

Reply:
Done.